src/sagemaker/feature_store/feature_group.py

```diff
@@ -599,6 +608,11 @@ def ingest(
         They can also be found from the IngestionManagerPandas' ``failed_rows`` function after
         the exception is thrown.
 
+        `profile_name` argument is an optional one. It will use the default credentialif None is
```

Comment:
a minor correction for adding space here: `credential if`

Reply:
Sure.